std_msgs/Header header
uint16 acc_x
uint16 acc_y
uint16 acc_z
uint16 acc_id

uint16 gyro_x
uint16 gyro_y
uint16 gyro_z
uint16 gyro_id